v0.3/tutorials/quickstart 2 button stopwatch.eve

```diff
@@ -0,0 +1,252 @@
+---
+layout: default
+title: Quickstart + Two Button Stopwatch
+---
+
+{% raw %}
+
+# Quickstart + Two Button Stopwatch
+
+This guide expands on the quickstart `Clock` page and shows you how to turn the clock into a two button stopwatch. The following code is meant to replace all the code from the original clock.
+
+## Implement system time
+
+````
+```
```

Comment:
we should probably update the code blocks to v0.4 style. This way, we don't have to talk about embedding them and showing nested code fences like this.